packages/preview-middleware-client/test/unit/adp/init.test.ts

```diff
@@ -42,12 +38,56 @@ describe('adp', () => {
         });
     });
 
+    beforeEach(() => {
+        rtaMock.getDefaultPlugins
+            .mockReturnValueOnce({
+                contextMenu: {
+                    addMenuItem: addMenuItemSpy
+                }
+            })
+            .mockReturnValueOnce({});
+    });
+
+    afterEach(() => {
+        initOutlineSpy.mockClear();
+        addMenuItemSpy.mockClear();
+        setPluginsSpy.mockClear();
+        sendActionMock.mockClear();
+    });
+
     test('init', async () => {
         const spyPostMessage = jest.spyOn(common, 'startPostMessageCommunication').mockImplementation(() => {
             return { dispose: jest.fn(), sendAction: jest.fn() };
         });
+        VersionInfo.load.mockResolvedValue({ version: '1.118.1' });
+
+        await init(rtaMock);
+
+        expect(initOutlineSpy).toBeCalledTimes(1);
+        expect(addMenuItemSpy).toBeCalledTimes(2);
+        expect(setPluginsSpy).toBeCalledTimes(1);
+
+        const callBackFn = spyPostMessage.mock.calls[0][1];
+
+        const payload = {
+            controlId: 'v2flex::sap.suite.ui.generic.template.ListReport.view.ListReport'
+        };
+
+        await callBackFn({
+            type: '[ext] add-extension-point',
+            payload
+        });
 
-        init(rtaMock);
+        expect(executeSpy).toHaveBeenCalledWith(payload.controlId, 'CTX_ADDXML_AT_EXTENSIONPOINT');
```

Comment:
This block of code is duplicated in two tests. It is testing the same thing two times, so maybe leave this block of code in the `init` test and remove it in the second test for the ui5 version notification.

Reply:
Done